WooCommerce/Classes/Tools/Logging/WooCrashLoggingStack.swift

```diff
@@ -146,7 +146,7 @@ class WCCrashLoggingDataProvider: CrashLoggingDataProvider {
         return .enabled(
             .init(
                 // FIXME: Is there a way to control this via feature flags?
```

Comment:
About usage of feature flags - on WCAndroid we recently disabled this feature as we were relying on Firebase Remote Config for that.

But for the future, I think it might be valuable to set performance sampling to use our in-house feature flag system (when it'll be integrated).